promql/lex.go

```diff
@@ -551,6 +561,44 @@ func lexInsideBraces(l *lexer) stateFn {
 	return lexInsideBraces
 }
 
+// lexValueSequence scans a value sequence of a series description.
+func lexValueSequence(l *lexer) stateFn {
+	switch r := l.next(); {
+	case r == eof:
+		return lexStatements
+	case isSpace(r):
+		lexSpace(l)
+	case r == '+':
+		l.emit(itemADD)
+	case r == '-':
+		l.emit(itemSUB)
+	case r == 'x':
+		l.emit(itemMUL)
```

Comment:
`itemMUL` is already used for `*`. The `x` is often just called "times", so how about `itemTimes`?